core/src/components/picker-column/picker-column.tsx

```diff
@@ -473,26 +452,17 @@ export class PickerColumn implements ComponentInterface {
   }
 
   render() {
-    const { color, disabled, isActive, numericInput } = this;
+    const { disabled, isActive, numericInput } = this;
     const mode = getIonMode(this);
 
-    /**
-     * exportparts is needed so ion-datetime can expose the parts
-     * from two layers of shadow nesting. If this causes problems,
-     * the attribute can be moved to datetime.tsx and set on every
-     * instance of ion-picker-column there instead.
-     * TODO FW-5580 remove exportparts
-     */
-
     return (
       <Host
-        exportparts={`${PICKER_ITEM_PART}, ${PICKER_ITEM_ACTIVE_PART}`}
```

Comment:
The column options exist in the light DOM now by default. For pickers in datetime, datetime will continue to expose shadow parts.